### PR TITLE
Practice some basics of HTML and CSS

### PR DESCRIPTION
**Pull request by oldjunior**

### CODE REVIEW

File: src/ex8_html-css-basics/css/task-04.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+.desc-list {
+  width: 50%;
+  height: 200px;
+  background-color: rgb(252, 165, 3);
+}
+li ul {
+  font-style: italic;
+}
+#ramm {
```
Comment (aneelia) [inline src/ex8_html-css-basics/css/task-04.css:23]: что такое ramm?

Comment (oldjunior) [reply src/ex8_html-css-basics/css/task-04.css:23]: stein:satisfied: The block's named after its content. New album ist Feuer!

Comment (aneelia) [reply src/ex8_html-css-basics/css/task-04.css:23]: f*ck yeah!

Comment (aneelia) [reply src/ex8_html-css-basics/css/task-04.css:23]: merged!

Comment (oldjunior) [reply src/ex8_html-css-basics/css/task-04.css:23]: :metal:

**Pull request merged**